quinn-proto/src/config/mod.rs

```diff
@@ -209,6 +209,10 @@ pub struct ServerConfig {
     /// rebinding. Enabled by default.
     pub(crate) migration: bool,
 
+    pub(crate) validation_token_lifetime: Duration,
+    pub(crate) validation_token_log: Option<Arc<dyn TokenLog>>,
```

Comment:
From the `TokenLog` docs, it sounds like the behavior of `None` here is equivalent to a log that always returns `Err`. Would it be simpler to omit the `Option` wrapper and provide a ZST impl with that behavior?